popSimManu.tex

```diff
@@ -302,14 +304,24 @@ \subsection*{The \stdpopsim Catalog}
 to complex models which include population splitting, migration, and archaic
 admixture \cite[e.g.,][]{ragsdale2019models}.
 
+\renewcommand{\arraystretch}{1.2}
 \begin{table}[t]
-\begin{footnotesize}
-\input{model_table.tex}
-\end{footnotesize}
+% Make sure the table is centered even though it's too wide
+\makebox[\textwidth][c]{
+    \begin{footnotesize}
+    \input{model_table.tex}
+    \end{footnotesize}
+}
 \caption{\label{tab:catalog}
-Details of the initial set of population models across four species.
-\textbf{NOTES: Column names are ID, name (or description maybe?),
-reference, CPU time, RAM}
+Initial set of demographic models and simple benchmarks.
```

Comment:
"Initial set of demographic models included in the Catalog and simple benchmarks"

Reply:
Done, thanks.